### PR TITLE
Initial Change to Website

### PR DESCRIPTION
**Pull request by sarahz916**
Created different sections of webpage. Finished introduction section and created other sections (i.e. Projects, Travels, Resume, and Contact) to add to.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,73 @@
...
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Sarah Zou's Portfolio</h1>
       <p>This is my portfolio.</p>
+      <p>A few quick links: </p>
+      <ul> 
+          <li><a href="#intro">Introduction </a> </li>
```
Comment (JKubs15) [inline portfolio/src/main/webapp/index.html]: Is there a reason these are indented differently?

**Review by JKubs15 — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (kkincy) [inline portfolio/src/main/webapp/index.html]: Nit: Should there be whitespaces after "Introduction" and each of these headings?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,73 @@
...
+     </p>
+     <p>
+          Here's a list of fun facts (because everyone is always asking for them in introductions):
+          <ul> 
+              <li> I play varsity women's water polo at Caltech. </li>
+              <li> Brunch is my favorite meal because it's two meals in one (
+                  also the word itself flows better than breakfast and lunch). </li>
+              <li> I've watch and finish way too many TV shows--Grey's Anatomy, 
+                  Gossip Girl, Money Heist, Hart of Dixie, White Collar--just to name a few.</li>
+              <li> I also really like reading books and listening to podcasts </li>
```
Comment (kkincy) [inline portfolio/src/main/webapp/index.html]: Nit: this sentence is missing a period.

**Review by kkincy — APPROVED** [review]
Looks good overall, left a few comments to test code reviews in Github. :)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sarahz916) [reply portfolio/src/main/webapp/index.html]: yep there should be I added those!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (sarahz916) [reply portfolio/src/main/webapp/index.html]: Added thanks!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sarahz916) [reply portfolio/src/main/webapp/index.html]: I wanted the list items to be within the unordered list brackets. I believe that is the accepted formatting for lists in HTML but I'm not 100% sure.

**Pull request merged**